R/LearnerClassifExtraTrees.R

```diff
@@ -0,0 +1,65 @@
+#' @title Classification ExtraTrees Learner
+#'
+#' @aliases mlr_learners_classif.extratrees
+#' @format [R6::R6Class] inheriting from [mlr3::LearnerClassif].
+#'
+#' @description
+#' A [mlr3::LearnerClassif] for a classification extratree implemented in extraTree::extraTree()] in package \CRANpkg{extraTree}.
+#'
+#' @export
+LearnerClassifExtraTrees = R6Class("LearnerClassifExtraTrees", inherit = LearnerClassif,
+  public = list(
+    initialize = function() {
+      ps = ParamSet$new(
+        params = list(
+          ParamInt$new(id = "ntree", default = 500L, lower = 1L, tags = c("train")),
+          ParamInt$new(id = "mtry", lower = 1L, tags = "train"),
+          ParamInt$new(id = "nodesize", default = 1L, lower = 1L, tags = "train"),
+          ParamInt$new(id = "numRandomCuts", default = 1L, tags = "train"),
+          ParamLgl$new(id = "evenCuts", default = FALSE, tags = "train"),
+          ParamInt$new(id = "numThreads", default = 1L, lower = 1L, tags = "train"),
+          ParamUty$new(id = "subsetSizes", tags = "train"),
+          ParamUty$new(id = "subsetGroups", tags = "train"),
+          ParamUty$new(id = "tasks", tags = "train"),
+          ParamDbl$new(id = "probOfTaskCuts", lower = 0, upper = 1, tags = "train"),
+          ParamInt$new(id = "numRandomTaskCuts", default = 1L, lower = 1L, tags = "train"),
+          ParamFct$new(id = "na.action", default = "stop", levels = c("stop", "zero", "fuse"), tags = "train")
+        )
+      )
+
+      super$initialize(
+        id = "classif.extratrees",
+        packages = "extraTrees",
+        feature_types = c("integer", "numeric"),
+        predict_types = c("response", "prob"),
+        param_set = ps,
+        properties = c("weights", "twoclass", "multiclass")
+      )
+    },
+
+    train_internal = function(task) {
+      pars = self$param_set$get_values(tags = "train")
+      data = task$data()
+      x = as.matrix(data[,task$feature_names, with=FALSE])
+      y = data[,task$target_names, with=FALSE][[1]]
```

Comment:
Space after `,` and around operators (`=`)

y = data[, task$target_names, with = FALSE][[1]]